### PR TITLE
remove unused methods from Utilities.cs

### PR DESCRIPTION
**Pull request by chriselion**
### Proposed change(s)

Remove unused methods from Utilities.cs

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [x] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: com.unity.ml-agents/Runtime/Utilities.cs [hunk 1]
```diff
@@ -75,83 +74,6 @@ public static int[] CumSum(int[] input)
...
-                if (sensors[i].GetCompressionType() == SensorCompressionType.None)
-                {
-                    numFloatObservations += sensors[i].ObservationSize();
-                }
-            }
-            return numFloatObservations;
-        }
-
 #if DEBUG
         internal static void DebugCheckNanAndInfinity(float value, string valueCategory, string caller)
```
Comment (vincentpierre) [inline com.unity.ml-agents/Runtime/Utilities.cs]:
```suggestion
        public static void DebugCheckNanAndInfinity(float value, string valueCategory, string caller)
```

File: com.unity.ml-agents/Runtime/Utilities.cs [hunk 2]
```diff
@@ -75,83 +74,6 @@ public static int[] CumSum(int[] input)
...
-        /// <summary>
-        /// Shifts list elements to the left by the specified amount (in-place).
-        /// <param name="list">
-        /// List whose elements will be shifted
-        /// </param>
-        /// <param name="shiftAmount">
-        /// Amount to shift the elements to the left by
-        /// </param>
-        /// </summary>
-        public static void ShiftLeft<T>(List<T> list, int shiftAmount)
```
Comment (vincentpierre) [inline com.unity.ml-agents/Runtime/Utilities.cs]: We did not have any tests for these?

Comment (chriselion) [reply com.unity.ml-agents/Runtime/Utilities.cs]: Nope 
image

File: com.unity.ml-agents/Runtime/Utilities.cs (hunk 1 above)
Comment (chriselion) [reply com.unity.ml-agents/Runtime/Utilities.cs]: Made the other methods in the class internal too.

**Pull request merged**